cassandra_check/metadata.csv

```diff
@@ -0,0 +1,3 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+cassandra.replication_availability,gauge,,,,Percentage of data available per keyspace times replication factor,+1,cassandra_check,available data
```

Comment:
we could use the name of the nodetool command as a prefix for future proofing, like:
`cassandra.status.replication_availability` or even `cassandra.nodetool.status.replication_availability`